.github/ISSUE_TEMPLATE/REPOSITORY_REQUEST_TEMPLATE.yaml

```
@@ -0,0 +1,123 @@
+---
+name: '📦Repository Request'
+description: 'Request a new repository with its publication target(s) in the OpenSearch Project.'
+title: '[Repository Request]'
+labels: ['repository-request']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to submit this Repository Request! Please note that the estimated time for the request to be resolved is 14-21 days from the time of the request.
+  - type: input
+    attributes:
+      description: |
+        Please note that, we are still awaiting nomination of initial BIG members per [OpenSearch Project Build Interest Group (BIG) Governance Proposal](https://github.com/opensearch-project/.github/issues/304).
+        As a result, requests for new GitHub Repo may experience delays.
+      label: |
+        Are you requesting a new GitHub Repo within opensearch-project GitHub Organization?
+        If not, please provide the URL of your current GitHub Repo.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      description: |
+        **(Enter 'N/A' in this section if you already have a GitHub Repo)**
+        1. Review the OpenSearch Software Foundation [Charter](https://foundation.opensearch.org/assets/media/OpenSearch%20Project%20Technical%20Charter%20Final%209-13-2024.docx.pdf).
+        2. Open a proposal issue with this [template](https://github.com/opensearch-project/.github/issues/new?template=PROPOSAL_TEMPLATE.md), see [examples](https://github.com/opensearch-project/.github/issues?q=is%3Aissue%20state%3Aopen%20%5BPROPOSAL%5D).
+        3. Attach the proposal issue url in above textarea.
+      label: 'GitHub Repo Proposal'
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      description: |
+        **(Enter 'N/A' in this section if you already have a GitHub Repo)**
+      value: |
+        1. What is the GitHub Repo name?
```

```suggestion
        1. What is the new GitHub repository name?
```

This could be a good place to link to conventions of some kind.